src/mono/CMakeLists.txt

```diff
@@ -290,6 +290,9 @@ elseif(TARGET_SYSTEM_NAME STREQUAL "Emscripten")
   if (CMAKE_BUILD_TYPE STREQUAL "Release")
     add_compile_options(-Os)
   endif()
+  if(NOT HOST_WASM)
+    set(GC_SUSPEND "default")
+  endif()
```

Comment:
Why is this needed ? It should only be hit for the cross compiler.

Reply:
yes this is for the cross compiler, I was hitting some issue otherwise.

Reply:
Wasn't this fixed by:
https://github.com/dotnet/runtime/pull/44093
?

Reply:
ah yeah, I'll check.

Reply:
removed it.